gabbytam_solution.txt

```diff
@@ -0,0 +1,93 @@
+//Problem set: https://github.com/Gabbytam/JS-Basic-Loops
+
+
+// 1. Get Even
+
+for(i=0; i<=200; i++){
+  if(i%2===0){
+    console.log(i);
+  }
+}
+
+//2. Excited Kitten
+
+i=1;
+while(i<=10){
+  console.log('Love me, pet me! HSSSSSS!');
+  let phrases= ["...human...why you taking pictures of me?...", "...the catnip made me do it...", "meow?", "...why does the red dot always get away..."];
+  let randNum= Math.floor(Math.random()*4);
+  if(i%2===0){
+    console.log(phrases[randNum]);
+  }
+  i++;
+}
```

Comment:
nice! you've simulated the functionality of a for loop with a while loop. pretty cool! 😄